app/helpers/dams_objects_helper.rb

```diff
@@ -688,6 +688,43 @@ def grabRestrictedText(data)
 
   end
 
+  #---
+  # Check to see if an object has a "metadataDisplay or localDisplay otherRights"
+  #
+  # @return An HTML string if an object has a "metadataDisplay or localDisplay otherRights", nil otherwise
+  #---
+
+  def grab_access_text(document)
+    result = nil
+    access_group = document['read_access_group_ssim'] # "public" > "local" > "dams-curator" == "dams-rci" == default
+    data = document['otherRights_tesim']
+    unless data.nil? && access_group.nil?
```

Comment:
you could probably do a `return result unless data && access_group` here if you want do drop the conditional